code/framework/object/src/main/java/io/cattle/platform/object/meta/impl/DefaultObjectMetaDataManager.java

```diff
@@ -612,6 +612,10 @@ public SchemaImpl postProcess(SchemaImpl schema, SchemaFactory factory) {
         Map<String, Relationship> result = new HashMap<String, Relationship>();
         Schema schema = schemaFactory.getSchema(type);
 
+        if (null == schema) {
```

Comment:
This check is required to prevent a NPE. In case of "POST" requests, the PostChildLinkHandler calls this code, and looks for schema of type "${REDIRECT_URL}", which returns null.